cli/kv.go

```diff
@@ -286,6 +286,33 @@ func runReverseScan(cmd *cobra.Command, args []string) {
 	showResult(rows)
 }
 
+var rangeDelCmd = &cobra.Command{
+	Use:   "rangedel [options] <startKey> <endKey>",
+	Short: "deletes key in range",
+	Long: `
+Deletes keys in range <startKey> and <endKey>. Both the keys are required
+for this command to run.
+`,
+	Run: runRangeDel,
+}
+
+func runRangeDel(cmd *cobra.Command, args []string) {
+	if len(args) != 2 {
+		cmd.Usage()
+		return
+	}
+	kvDB := makeDBClient()
+	if kvDB == nil {
+		return
+	}
+	startKey := proto.Key(unquoteArg(args[0], false))
```

Comment:
let's change these `false`s to `true`s; we don't want to allow system keys to be deleted. (see style in #2103)